repos/system_upgrade/common/models/packagemanagerinfo.py

```diff
@@ -17,3 +17,8 @@ class PkgManagerInfo(Model):
     In case the value is empty string, it means the file exists but it is empty. In such a case the
     original configuration is obviously broken.
     """
+
+    dnf_proxy = fields.Nullable(fields.String())
+    yum_proxy = fields.Nullable(fields.String())
```

Comment:
Is it important where does the proxy comes from? For the purpose of the changes introduced in this PR, IMHO it does not matter where the proxy was detected, it is important that the source system was using proxy to access new content (package updates), which could cause problems in the `target_userspace_creator`. If it the source of the proxy is important, then the checks in `target_userspace_creator` should check for both - the source system has used a means of reaching content through a proxy, and we do not posses those means which could cause the dnf error.